ocs_ci/ocs/ocp.py

```diff
@@ -358,15 +359,21 @@ def get_resource_status(self, resource_name):
             str: The status returned by 'oc get' command not in the 'yaml'
                 format
         """
-        status_index = None
+        # Get the resource in str format
         resource = self.get(resource_name=resource_name, out_yaml_format=False)
+        # get the list of titles
+        titles = re.sub('\s{2,}', ',', resource)  # noqa: W605
+        titles = titles.split(',')
+        # Get the index of 'STATUS'
+        status_index = titles.index('STATUS')
         resource = shlex.split(resource)
-        for idx, i in enumerate(resource):
-            if i.isupper() and i == 'STATUS':
-                status_index = idx
-            if not i.isupper():
-                break
-        resource_info = [i for i in resource if not i.isupper()]
+        # Get the values from the output including access modes in capital
+        # letters
```

Comment:
Could we add explicit note that we basically assume that all content lines from the output are not using all caps (with exception of access mode), because that is the way we figure out what is a content and what a title? This won't work if we use capital letters only in a name of storageclass, pv or a namespace.